lib/src/calendar.dart

```diff
@@ -52,4 +66,24 @@ abstract class Calendar implements Comparable<Calendar> {
   DateTime toDateTimeLocal();
 
   Calendar copy();
+
+  bool operator <(Calendar other);
+
+  bool operator <=(Calendar other);
+
+  bool operator >=(Calendar other);
+
+  bool operator >(Calendar other);
+
+  Calendar operator +(CalendarDuration<Calendar> duration);
+
+  /// If you subtract another [Calendar] from this, the fucntion
+  /// will return the [CalendarDuration] between the two dates.
+  /// 
+  /// If you subtract a [CalendarDuration] from this, the function
+  /// will return a new Calendar that is the given distance back
+  /// in time.
+  dynamic operator -(dynamic other);
```

Comment:
This seems confusing.  I would rather just have named methods that are clearer about what you're adding or subtracting.

Reply:
Fair enough. Although I prefer the more compact syntax of a simple operator, using only the named methods is also fine. `addCalendarDuration()` and `calculateDurationToDate()` implement this functionality so we can just delete the add and subtract operators.

Reply:
I've added another commit which removes these operators.